pkg/datapath/loader/base.go

```diff
@@ -400,6 +387,33 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 	args[initArgHostDev1] = hostDev1.Attrs().Name
 	args[initArgHostDev2] = hostDev2.Attrs().Name
 
+	if option.Config.EnableIPv6 {
+		args[initArgIPv6NodeIP] = node.GetIPv6().String()
+		// Docker <17.05 has an issue which causes IPv6 to be disabled in the initns for all
+		// interface (https://github.com/docker/libnetwork/issues/1720)
+		// Enable IPv6 for now
+		sysSettings = append(sysSettings,
+			sysctl.Setting{Name: "net.ipv6.conf.all.disable_ipv6", Val: "0", IgnoreErr: false})
+
+		ciliumHostV6Addrs, err := netlink.AddrList(hostDev1, netlink.FAMILY_V6)
+		if err != nil {
+			return err
+		}
+		for _, addr := range ciliumHostV6Addrs {
+			// Delete stale cilium_host IPv6 to make sure downgrade from 1.14 to 1.13 works:
+			// https://github.com/cilium/cilium/issues/25938
+			if addr.Scope == int(netlink.SCOPE_UNIVERSE) && addr.IP.String() == args[initArgIPv6NodeIP] {
```

Comment:
Last nit - let's add a info msg that we are removing that IP addr. This would give a visibility to users which manually added IPv6 addrs.

Reply:
Actually, I implemented the logic totally wrong :rofl: it should be `addr.IP.String() != args[initArgIPv6NodeIP] ` but I wrote `==`.
And of course, info log :+1: